### ISSUE TITLE
Can't access titles in non-public Liked Lists from Private Trakt accounts

### ISSUE THREAD

**Issue by daherlihy** [kodifitzwell/repo#9]
I've just noticed that I can no longer access non-public Liked Lists on Private Trakt accounts.

For further context, there's no issue accessing Liked Lists with the privacy set to Public.

NOTE - I'm working with the dev for FenLight on the same. If you want further context and explanation on how Trakt lists work, I've put some details in here: https://github.com/Tikipeter/tikipeter.github.io/issues/24

**Comment by kodifitzwell** [comment]
try it and post a log please.  there should be some logging trakt is send back as to the reason.

**Comment by daherlihy** [comment]
Im actually going to cancel this and just work with public lists which appear to work fine - I appreciate that there's a lot of hassle I'm causing here for you. Thanks again for all that you do :)

**Comment by kodifitzwell** [comment]
as far as I can tell from the api docs there are two ways to access list items.

    users/id/lists/list_id/items
    
    id (username) has to be slugified which generally isn't a problem but I did come across one
    list_id can be either trakt id (number) or the slugified list name
      
the docs say "OAuth Optional", but if the list is personal and private it does require authorization.  also I did message @tikipeter on my code update to ensure the id is slugified as currently it wasn't.  and I think I'm going to swap out the slugified list name to trakt id also.

    lists/id/items
    
    id is the trakt id (number).

switching to this was my first attempt to fix the non slugified "id" in the above method.  which worked until a user tries to access a personal and private list, even with auth it did not work.  so I switched back to the above method and fixed the code to pick up the slugified id (username).

the docs make no mention about accessing private/liked lists on another private account.  there is a trakt/api-help github for those kind of questions I guess.  maybe ask there if its possible and/or what the auth requirements are and get back to either tikipeter or I.
